dev_scripts/translation.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-console */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+const fs = require('fs');
+
+const localizationFiles = [];
+const nextLine = /{nextLine}/g;
+const selectedLanguage = process.argv[3];
+const newFileFolder = './src/localization/translations/';
+const message =
+  '\nMore information about this script here: https://github.com/openmsupply/mobile/wiki/Scripts\n';
+
+// Reads all JSON files in localization files and save them in localizationFiles array
+const filesFolder = './src/localization/';
+fs.readdirSync(`./${filesFolder}`).forEach(file => {
+  if (file.split('.').pop() === 'json') {
+    localizationFiles.push({
+      fileContent: require(`../${filesFolder}${file}`),
+      fileName: file.replace('.json', ''),
+    });
+  }
+});
+
+// Exports all strings from the selectedLanguage to CSV
+// csvData keeps all the content in csv formal
+const exportTranslation = () => {
+  let csvData = '';
+  localizationFiles.forEach(localizationFile => {
+    const valuesObject = {};
+    const selectedValuesObject = {};
+    Object.entries(localizationFile.fileContent.gb).forEach(([key, value]) => {
+      valuesObject[key] = value.replace(/\n/gi, '{nextLine}');
+      selectedValuesObject[key] = localizationFile.fileContent[selectedLanguage][key]
+        ? localizationFile.fileContent[selectedLanguage][key].replace(/\n/gi, '{nextLine}')
+        : '';
+      csvData = `${csvData} ${localizationFile.fileName},${key},"${valuesObject[key]}","${selectedValuesObject[key]}"\n`;
```

Comment:
i don't see valuesObject[key] and selectedValuesObject[key] needing to be objects

Reply:
Agree, I took them out and I directly use the readed values.

```
    Object.entries(mainLanguageContent).forEach(([key, value]) => {
      value = value.replace(/\n/gi, '{nextLine}');
      if (selectedLanguageContent[key]) {
        selectedLanguageContent[key] = selectedLanguageContent[key].replace(/\n/gi, '{nextLine}');
      } else selectedLanguageContent[key] = '';
      csvData = `${csvData} ${fileName},${key},"${value}","${selectedLanguageContent[key]}"\n`;
    });
```